docs/product-user-guide/src/main/asciidoc/assets-unlocking-proc.adoc

```diff
@@ -8,17 +8,20 @@ However, you can force unlock an asset if you need to edit a file that is locked
 .Procedure
 
 . Go to *Menu* -> *Design* -> *Projects* and click the project name.
-. Select the asset from the list to open the asset editor in the *Project Explorer*.
+. Select the asset from the list to open the asset designer in the *Project Explorer*.
 . Go to *Overview -> Metadata* and view the *Lock Status*.
 +
+.Unlock metadata view
+image::unlock-lock.png[Unlock Metadata View]
++
 If the asset is already being edited by another user, the following will be displayed in the *Lock status*
```

Comment:
There is also a lock icon next to the asset name once the asset is locked.

Reply:
Added info.